libraries.txt

```diff
@@ -13,3 +13,4 @@ numpy==1.19.3
 pandas==1.1.4
 scikit-learn==0.23.2
 scipy==1.5.4
+pyyaml==5.3.1
```

Comment:
Not sure we should be adding to the competitor libraries because our internal scripts need something.